homeassistant/components/amberelectric/sensor.py

```diff
@@ -75,8 +80,8 @@ def native_value(self) -> str | None:
         interval = self.coordinator.data[self.entity_description.key][self.channel_type]
 
         if interval.channel_type == ChannelType.FEED_IN:
-            return round(interval.per_kwh, 0) / 100 * -1
-        return round(interval.per_kwh, 0) / 100
+            return str(format_cents_to_dollars(interval.per_kwh) * -1)
+        return str(format_cents_to_dollars(interval.per_kwh))
```

Comment:
We don't have to cast it to a string. A `float` is a perfectly valid `StateType`.

Reply:
@frenck - I thought that was the case, although the developer docs for the `sensor` entity indicates `native_value` should be string: https://developers.home-assistant.io/docs/core/entity/sensor/

Update to remove the cast to string for the PR